lib/client.js

```diff
@@ -91,43 +92,52 @@ function _connect (client) {
             err = new Error('aborted')
           }
 
-          callback()
```

Comment:
Don't allow more requests until whole response has been processed

Reply:
This was done to increase throughput and utilization. Essentially if we are starting to receive data, we can send more as most of the response has been computed on the server.

Reply:
> This was done to increase throughput and utilization. Essentially if we are starting to receive data, we can send more as most of the response has been computed on the server.

That's still achievable by using a higher pipelining factor? This goes back to, https://github.com/mcollina/undici/issues/77.

Reply:
note, that this PR only changes the case where the response is aborted.